src/main/kotlin/no/nav/familie/tilbake/api/dto/HentBehandlingResponsDto.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.familie.tilbake.api.dto
+
+import no.nav.familie.tilbake.behandling.domain.Behandlingsresultatstype
+import no.nav.familie.tilbake.behandling.domain.Behandlingsstatus
+import no.nav.familie.tilbake.behandling.domain.Behandlingstype
+import no.nav.familie.tilbake.behandling.domain.Fagsaksstatus
+import no.nav.familie.tilbake.behandling.domain.Ytelsestype
+import no.nav.familie.tilbake.integration.pdl.internal.Kjønn
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class HentBehandlingResponsDto(val behandling: BehandlingDto,
```

Comment:
HentBehandlingResponsDto blir litt omfattende for meg: Hva med BehandlingsresponsDto

Reply:
endret navn til BehandlingsresponsDto